### PR TITLE
Allow manipulation of nncp deployment timeout

### PR DESCRIPTION
**Pull request by AdiZav**
##### Short description:
We've been having nncp deployment failures due to short success timeout

### CODE REVIEW

File: ocp_resources/node_network_configuration_policy.py [hunk 1]
```diff
@@ -385,7 +385,7 @@ def wait_for_status_success(self):
         # if we get here too fast there are no conditions, we need to wait.
         self.wait_for_configuration_conditions_unknown_or_progressing()
 
-        samples = TimeoutSampler(wait_timeout=480, sleep=1, func=self.status)
+        samples = TimeoutSampler(wait_timeout=540, sleep=1, func=self.status)
```
Comment (myakove) [inline ocp_resources/node_network_configuration_policy.py:388]: please get wait_timeout arg with default 480 and pass it when needed.

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/node_network_configuration_policy.py (hunk 1 above)
Comment (AdiZav) [reply ocp_resources/node_network_configuration_policy.py:388]: ACK

**Review by myakove — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Comment by myakove** [comment]
Please rename the title, this is no longer the case.

**Comment by AdiZav** [comment]
Sure

**Review by rnetser — APPROVED** [review]

**Comment by AdiZav** [comment]
!verified

**Pull request merged**